packages/fury-adapter-oas3-parser/test/unit/parser/oas/parseResponseObject-test.js

```diff
@@ -12,38 +12,19 @@ describe('Response Object', () => {
   });
 
   it('returns an HTTP response element', () => {
-    const response = new namespace.elements.Member('200', {
+    const response = new namespace.elements.Object({
       description: 'response',
     });
     const result = parse(context, response);
 
     expect(result.length).to.equal(1);
     const httpResponse = result.get(0);
     expect(httpResponse).to.be.instanceof(namespace.elements.HttpResponse);
-    expect(httpResponse.attributes.get('statusCode').toValue()).to.be.equal('200');
-  });
-
-  describe('status code', () => {
```

Comment:
This test was removed, but I don't see a replacement

Reply:
There is already a test for this behaviour in parseResponsesObject-test.js, you can see it in the diff below.